storage/errors.go

```diff
@@ -0,0 +1,10 @@
+package storage
+
+import "errors"
+
+var (
+	// ErrNotExists may be thrown by docs. E.g., save an unknown document.
+	//
+	// Since: 2.2.2
```

Comment:
This should be Since 2.3 - we don't add new API in patch releases (plus v2.3.0 is the next release). As you can see the api docs are grouped with just 2 digits to match this design.